### DISCUSSION TITLE
Spacing between icon and text on badge component

### DISCUSSION THREAD

**Discussion by rosemaryluo** [twilio-labs/paste#2553] [Q&A] [answered]
Hello! Spacing in our implementation seems smaller than in the figma component, wondering whether this is an issue with the component or how we've implemented. 

Implementation (https://www.stage.twilio.com/code-exchange behind split)
<img width="147" alt="image" src="https://user-images.githubusercontent.com/87049408/180558307-c1a76c43-a9b9-4cce-9191-5d55d37826ad.png">

[Figma](https://www.figma.com/file/wenSZVIkPx2dViALatMb91/CE-React-Migration-(Supplemental)?node-id=245%3A82556)
<img width="758" alt="image" src="https://user-images.githubusercontent.com/87049408/180556910-bcffea5e-ee88-46e0-bbc0-70a0ffb34ccc.png">

**Comment by shleewhite** [comment, marked as answer]
Hi @rosemaryluo,

Thank you for catching this! Figma is correct, so I made a PR to fix the Badge component: https://github.com/twilio-labs/paste/pull/2554

I will follow up on this thread when it is merged!

**Reply by shleewhite** [reply]
@rosemaryluo the badge fix is now live in `@twilio-paste/core@14.2.0`!
